### PR TITLE
Support CUDA 11.5

### PR DESCRIPTION
**Pull request by takagi**

### CODE REVIEW

**Comment by takagi** [comment]
Images are not ready.

File: dist_config.py [hunk 1]
```diff
@@ -10,8 +10,7 @@
 # See descriptions of WHEEL_LINUX_CONFIGS for details.
 SDIST_CONFIG = {
     'image': 'nvidia/cuda:10.2-devel-centos6',
-    # This image contains cuDNN and NCCL.
-    'verify_image': 'nvidia/cuda:11.4.2-cudnn8-devel-{system}',
+    'verify_image': 'nvidia/cuda:11.5.0-devel-{system}',
```
Comment (takagi) [inline dist_config.py:13]: I can see no CUDA 11.5 image contains cuDNN and NCCL for now in https://gitlab.com/nvidia/container-images/cuda

**Review by kmaehashi — CHANGES_REQUESTED** [review]

**Comment by kmaehashi** [comment]
Just installed CUDA 11.5 on FlexCI Windows image.
/test

**Review by kmaehashi — APPROVED** [review]

**Pull request merged**